.github/experience.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Experience</title>
+</head>
+<body>
+    <div class="content">
+        <header>
+            <div class="menuBox">
+                <div class="dropdown">
+                    <button class="dropbtn">Dropdown</button>
+                    <div class="dropdown-content">
+                      <a href="bio.html">Bio</a>
+                      <a href="portfolio.html">Portfolio</a>
+                      <a href="index.html">Home</a>
+                    </div>
+                </div>
+            </div>
+        </header>
+        <main>
+            <div class="interestStatementBox">
+                <h1>Work Experience</h1>
+            </div>
+            <div class="interests3">
+                     
+               <p>HTML <br> CSS, JS</p>
+                <ul>
+                     
+                </ul>                
+                <img src="html.webp" alt="Solar">
+
+            </div>
+            <div class="interestStatementBox">
+                <h1>Education</h1>
+            </div>
+            <div class="interests3">
+               <p>Parkway south High School <br> 08/2015 - 05/2019</p>
+                <ul>
+                    <li>Lorem ipsum dolor sit amet, probo idque postea ea qui, diam petentium</li> 
```

Comment:
Would love to see some more details here! Lorem obviously makes sense as a placeholder when necessary, but see if you can include actual info when possible!